community/README.md

```diff
@@ -3,3 +3,8 @@
 Welcome to the Argo community!
 
 Argo is an open, community driven project to make it easy to use Kubernetes for getting useful work done. [This document](https://github.com/argoproj/argoproj/blob/main/community/README.md) describes the organizational structure of the Argo Community including the roles, responsibilities and processes that govern Argo projects and community.
```

Comment:
```suggestion
Argo is an open, community driven project to make it easy to use Kubernetes for getting useful work done.
The [Argoproj Community README](https://github.com/argoproj/argoproj/blob/main/community/README.md) describes the organizational structure and governance of Argo projects including roles, responsibilities, and processes.
```

Might as well update this part of the doc while at it too:
- Avoid ambiguous "this document", [per k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#links)
- Prefer 1 sentence per line of markdown [per style guide](https://argoproj.github.io/argo-workflows/doc-changes/)
- Simpler and more direct language, [per k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)